### PR TITLE
Added dumped onboard blobs validation

### PR DESCRIPTION
**Pull request by iam-theKid**

### CODE REVIEW

File: sshrd.sh [hunk 1]
```diff
@@ -43,6 +43,39 @@ elif [ "$1" = 'dump-blobs' ]; then
...
+        read -p "Insert the iOS version (device current version) to validate dumped blobs: " blobsVersion
+        while [[ -z $blobsVersion ]]; do
+            echo "\niOS version cannot be blank!"    
+            read -p "Insert the iOS version (device current version) to validate dumped blobs: " blobsVersion
+        done
+    fi
+    otaBuildID=$(curl -sL "https://api.ipsw.me/v4/device/$deviceid?type=ipsw" | "$oscheck"/jq '.firmwares | .[] | select(.version=="'$blobsVersion'") | .buildid' --raw-output)
+    otaURL=$(curl -sL "https://api.ipsw.me/v4/device/$deviceid?type=ota" | "$oscheck"/jq '.firmwares | .[] | select(.buildid=="'$otaBuildID'") | .url' --raw-output)
+    echo "Downloading $deviceid $blobsVersion OTA BuildManifest.plist..."
+    pzb -g AssetData/boot/BuildManifest.plist $otaURL
```
Comment (itsnebulalol) [inline sshrd.sh:64]: Use `"$oscheck"/pzb`

Comment (iam-theKid) [reply sshrd.sh]: fixed

**Review by iam-theKid — COMMENTED** [review]
Added dumped blobs validation against OTA BuildManifest.plist if failed, run validation and IPSW BuildManifest.plst (sometimes dumped blobs are invalid against OTA but valid against IPSW)

**Review by iam-theKid — COMMENTED** [review]
fixed typo

**Pull request closed without merging**